routesrv/routesrv.go

```diff
@@ -42,12 +64,24 @@ func New(opts skipper.Options) (*RouteServer, error) {
 		return nil, err
 	}
 
-	b := &eskipBytes{tracer: tracer, now: time.Now}
-	bs := &eskipBytesStatus{b: b}
-	handler := http.NewServeMux()
-	handler.Handle("/health", bs)
-	handler.Handle("/routes", b)
-	handler.Handle("/metrics", promhttp.Handler())
+	b := &eskipBytes{
+		tracer:  tracer,
+		metrics: m,
+		now:     time.Now,
+	}
+	bs := &eskipBytesStatus{
+		b: b,
+	}
+	mux := http.NewServeMux()
+	mux.Handle("/health", bs)
+	mux.Handle("/routes", b)
+	mux.Handle("/metrics", metricsHandler)
+	mux.Handle("/metrics/", metricsHandler)
+
+	if opts.EnableProfile {
+		mux.Handle("/debug/pprof", metricsHandler)
+		mux.Handle("/debug/pprof/", metricsHandler)
+	}
```

Comment:
We should serve these from another port (support listener) since we allow user access to /routes

Reply:
yes, true what do you think about splitting similar to skipper itself?
/routes and /health would be the main listener and /metrics and /debug on supportListener.

Reply:
Yeah, I think we should have it the same as in skipper for simplicity

Reply:
this was done